lib/shippo/api.rb

```diff
@@ -44,9 +44,7 @@ def debug?
   end
 end
 
-Shippo.dir('shippo/api')
-Shippo.dir('shippo/api/transformers')
-Shippo.dir('shippo/api/extend')
-Shippo.dir('shippo/model')
-
-
+Dir[File.dirname(__FILE__) + '/api/*.rb'].each {|file| require file }
```

Comment:
This seems like an awful lot of repetitive code.

Can you please instead provide more details on why `require_dir` was an issue?